src/Order.java

```diff
@@ -50,9 +52,13 @@ public void printOrderDetails() {
         System.out.println("Order Price: $" + orderPrice);
     }
 
-    public double calculatePrice(String subscription) {
-        priceCalculator = new PriceCalculator(subscription);
+    public double calculateTotalPrice() {
```

Comment:
Remove this function from Order.java. We should only be using the function from priceCalculator to solve the smell.